addons/pos_sale/models/account_move.py

```diff
@@ -0,0 +1,24 @@
+from odoo import models, _
+
+
+class AccountMove(models.Model):
+    _inherit = 'account.move'
+
+    def reflect_cancelled_sol(self, isCancelled):
+        if self.env.user.has_group('point_of_sale.group_pos_user'):
+            for invoice in self:
+                for pos_order_line in invoice.pos_order_ids.mapped('lines'):
+                    if isCancelled and "(Cancelled)" not in pos_order_line.sale_order_line_id.name:
+                        pos_order_line.sale_order_line_id.name = _("%(line_description)s (Cancelled)", line_description=pos_order_line.sale_order_line_id.name)
```

Comment:
```suggestion
                        name = _("%(old_name)s (Cancelled)", old_name=pos_order_line.sale_order_line_id.name)
                        pos_order_line.sale_order_line_id.name = name
```

Reply:
Try to reduce line length

Reply:
Ok, i understood,
Updated the code accordingly,
Thanks.